components/map/index.js

```diff
@@ -1,17 +1,22 @@
+/* eslint-disable class-methods-use-this */
 /* eslint-disable react/no-unused-state */
 import React, { Component } from 'react';
-import MapView, { Polygon, PROVIDER_GOOGLE } from 'react-native-maps';
+import { View } from 'react-native';
+import MapView, { Polygon, Polyline, PROVIDER_GOOGLE } from 'react-native-maps';
 import buildings from '../../assets/polygons/polygons';
 import styles from './styles';
 
 export default class TheMap extends Component {
   constructor(props) {
     super(props);
+    const waypoints = '';
```

Comment:
Is this really needed? Could you not just initialize like this instead: `coordinates: ''`

Reply:
Nice catch! I will fix that

Reply:
in commit [#18] Fix minor coding issues (useless variables and console.log)